dali/test/python/auto_aug/test_trivial_augment.py

```diff
@@ -29,10 +29,11 @@
 images_dir = os.path.join(data_root, 'db', 'single', 'jpeg')
 
 
-@params(*tuple(enumerate(itertools.product((True, False), (True, False), (None, 0),
-                                           (True, False)))))
+@params(*tuple(
+    enumerate(
+        itertools.product(("cpu", "gpu"), (True, False), (True, False), (None, 0), (True, False)))))
```

Comment:
Does it really have to be a product? How long does this test take?
I'm not saying to change it, but rather to raise awareness about the potential impact of overusing Cartesian products of arguments.

Reply:
Sure, I try to keep them from exploding. For example the batch_size and the number of magnitdue bins are function of the params above not a factor in the product.
Here, the test is rather fast and simple. Its point is excaly to make sure that the pipeline builds and runs with all the combinations of the major parameters.